File: Data Structures & Algorithms/two-integer-sum/submission-1.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target)
{
    Dictionary<int, int> map = new Dictionary<int, int>();
    for (int i = 0; i < nums.Length; i++)
    {
        int difference = target - nums[i];
        if (!map.ContainsKey(nums[i]))
        {
            map[difference] = i;
        }
        else
        {
            return new int[] {map[nums[i]], i};
        }

    }
    return new int[] {};
}
}

File: Data Structures & Algorithms/is-anagram/submission-1.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        nums = [0] * 26
        for i in s:
            nums[ord(i) - ord('a')] += 1
        for i in t:
            nums[ord(i) - ord('a')] -= 1
        for i in nums:
            if i != 0: return False
        
        return True